### PR TITLE
Stipnek

### PR DESCRIPTION
**Pull request by Stipnek**
все раскидал по папочкам, руты некоторые автоматически не поменялись но я сделал. Выдает ошибку на страницах Sign-In и registration из-за firebase, это не трогал.

### CODE REVIEW

File: src/components/Logbar/Logbar.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { BsPencilSquare } from "react-icons/bs";
+import { BsLockFill } from "react-icons/bs";
+
+import "./Logbar.scss";
+
+const Logbar = () => {
+  const [selectedOption, setSelectedOption] = useState("login");
+
+  const handleOptionChange = (event) => {
+    setSelectedOption(event.target.value);
```
Comment (DenisTagaev) [inline src/components/Logbar/Logbar.jsx]: Consider adding here useNavigate to change the page depending on the selected option

File: src/components/Logbar/Logbar.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <div className="logBarMainContainer">
+      <label>
+        <input
+          type="radio"
+          name="option"
+          value="login"
+          checked={selectedOption === "login"}
+          onChange={handleOptionChange}
+        />
+        <span className="checkmark">
```
Comment (DenisTagaev) [inline src/components/Logbar/Logbar.jsx]: Not descriptive enough, checkmark of what? Element? Page?

File: src/components/Logbar/Logbar.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      </label>
+      <label>
+        <input
+          type="radio"
+          name="option"
+          value="register"
+          checked={selectedOption === "register"}
+          onChange={handleOptionChange}
+        />
+        <span className="checkmark">
```
Comment (DenisTagaev) [inline src/components/Logbar/Logbar.jsx]: same here, add common class for 3 checkmarks but describe where it belongs. Apply same styles to 3 spans like:
checkmark-login, checkmark-register, checkmark-restorePass (for example)

File: src/components/Logbar/Logbar.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          checked={selectedOption === "forgot"}
+          onChange={handleOptionChange}
+        />
+        <span className="checkmark"><BsLockFill className="logIcons"/>Forgot Password?</span>
+      </label>
+    </div>
+  );
+};
+
+export default Logbar;
```
Comment (DenisTagaev) [inline src/components/Logbar/Logbar.jsx:74]: Good consistency with tabulation, keep it this way. Consider adding comments to the code

File: src/components/Logbar/Logbar.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      background-color: $mainDark;
+      color: #fff;
+    }
+    &:not(:checked) + .checkmark {
+      color: #000;
+    }
+
+  }
+
+  .checkmark {
```
Comment (DenisTagaev) [inline src/components/Logbar/Logbar.scss]: Read the logbar.jsx comment regarding classes here

File: src/components/Sidebar/Sidebar.jsx [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
-import { Navbar } from './Navbar';
-import { Search } from './Search';
-import { Chats } from './Chats';
+import { Navbar } from '../Navbar/Navbar';
+import { Search } from '../Search/Search';
+import { Chats } from '../Chats/Chats';
```
Comment (DenisTagaev) [inline src/components/Sidebar/Sidebar.jsx:5]: Elements exported by default, no need to parse them

File: src/index.scss [hunk 7]
```diff
@@ -5,18 +5,36 @@ $secondary: #75E6DA;
...
+  background-color: $primary;
+}
+
+@mixin logIcons {
+  display: block;
+  font-size: 28px;
+  margin: 0 auto;
+}
+
+@mixin inputBox {
```
Comment (DenisTagaev) [inline src/index.scss]: A lot of inputs, not each one will be styled the same, specify for which page

File: src/pages/Sign-In/Login.scss [hunk 8]
```diff
@@ -6,10 +6,16 @@
...
     align-items: center;
     justify-content: center;
+    gap: 2%;
 
     .loginWrap {
-        background-color: $mainLight;
-        padding: 1.8% 5.4%;
+        @include inputBox;
+
+        .title {
```
Comment (DenisTagaev) [inline src/pages/Sign-In/Login.scss:17]: title what? King of GB?

**Review by DenisTagaev — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/Sidebar.jsx (hunk 6 above)
Comment (Stipnek) [reply src/components/Sidebar/Sidebar.jsx:5]: this is not mine=) i will change it

File: src/components/Logbar/Logbar.jsx (hunk 1 above)
Comment (Stipnek) [reply src/components/Logbar/Logbar.jsx]: useLocation works here, as you showed

File: src/components/Logbar/Logbar.jsx (hunk 2 above)
Comment (Stipnek) [reply src/components/Logbar/Logbar.jsx]: changed on logCheckmark

File: src/components/Logbar/Logbar.jsx (hunk 4 above)
Comment (Stipnek) [reply src/components/Logbar/Logbar.jsx:74]: ok

File: src/index.scss (hunk 7 above)
Comment (Stipnek) [reply src/index.scss]: inputBox is used for all pages

File: src/pages/Sign-In/Login.scss (hunk 8 above)
Comment (Stipnek) [reply src/pages/Sign-In/Login.scss:17]: title className is turned into mixin

**Review by DenisTagaev — APPROVED** [review]

**Pull request merged**